Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -1210,3 +1210,32 @@ instance [hα : Nonempty α] : IsEmpty (SuccOrder (WithBot α)) :=
 end Succ
 
 end WithBot
+
+section OrderIso
+
+variable {X Y : Type*} [Preorder X] [Preorder Y]
+
+-- See note [reducible non instances]
+/-- `SuccOrder` transfers across equivalences between orders. -/
+protected abbrev SuccOrder.of_orderIso [SuccOrder X] (f : X ≃o Y) : SuccOrder Y where
```

Comment:
Since it carries data, I think that it should use `lowerCamelCase`.
```suggestion
protected abbrev SuccOrder.ofOrderIso [SuccOrder X] (f : X ≃o Y) : SuccOrder Y where
```